build.gradle.kts

```diff
@@ -68,34 +62,30 @@ subprojects {
   }
 
   dependencies {
+    implementation("org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE")
+
```

Comment:
Can you use `libs...` here instead of literal strings, as you're doing in the other parts of the code?

Reply:
Thanks for catching the missing alias. Fixed!